src/Layer/OrientedImageLayer.js

```diff
@@ -130,8 +130,9 @@ class OrientedImageLayer extends GeometryLayer {
             let i = 0;
             for (const pano of this.panos) {
                 // set position
-                coord.set(pano.crs, pano.vertices[0], pano.vertices[1], pano.vertices[2]);
-                pano.position = coord.xyz();
+                coord.crs = pano.crs;
+                coord.set(pano.vertices[0], pano.vertices[1], pano.vertices[2]);
```

Comment:
`set` doesn't work with Array? 

it may be well to code the function `setFromArray(array, offset = 0)`, WDYT?

Reply:
It does work with an array, I just didn't change it correctly here. But I'm all for a `setFromArray` as well as `setFromVector3`, it would avoid having conditions in `set`. WDYT ?

Reply:
ok to write `setFromArray`